modules/gathering-bootstrap-diagnostic-data.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// *support/troubleshooting-installations.adoc
+
+[id="gathering-bootstrap-diagnostic-data_{context}"]
+= Gathering bootstrap node diagnostic data
+
+When experiencing bootstrap related issues, gather `bootkube.service` journald unit logs and container logs from the bootstrap node.
+
+.Prerequisites
+
+* The `ssh-agent` process is active on your computer, and you provided both the `ssh-agent` process and the installation program with the same SSH key.
+* If you tried to install a cluster on infrastructure that you provisioned, you must have the fully qualified domain names of the bootstrap node.
+
+.Procedure
+
+. If you have access to the bootstrap node's console, monitor the console until the node reaches the login prompt.
+
+. Verify the ignition file configuration:
```

Comment:
Double check - I think that "Ignition" is supposed to be capitalized, but not 100% sure (check throughout rest of PR if it is)